src/routes/api/comments.route.js

```diff
@@ -34,6 +36,43 @@ const router = express.Router();
  *       '403':
  *         description: User not allowed to comment on this request
  */
-router.post('/requests/:requestId/comment', verifyUser, validation, catchErrors(CommentController.postComment));
+router.post('/requests/:requestId/comment', verifyUser, validation, catchErrors(postComment));
+
 
+/**
+ * @swagger
+ *
+ * /comment/:commentId/delete:
+ *   delete:
+ *     summary: Delete comment
+ *     description: Set the comment to be invisible from the comments thread
+ *     tags:
+ *       - Comments
+ *     produces:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               status:
+ *                 type: string
+ *               message:
+ *                 type: string
+ *               data:
+ *                 type: object
+ *                 properties:
+ *                   id:
+ *                     type: integer
+ *                   comment:
+ *                     type: string
+ *                   isVisible:
+ *                     type: boolean
+ *     responses:
+ *       200:
+ *         description: Successfully deleted
+ *       404:
+ *         description: Comment not found
+ *       403:
+ *         description: Unauthorized user
+ */
+router.patch('/comment/:commentId/delete', verifyUser, catchErrors(deleteComment));
```

Comment:
> I think you should rename this route to:
> router.delete('/comment/:commentId', verifyUser, catchErrors(deleteComment));
> 
> That is, use the 'router.delete' function since it is a deletion endpoint.

Thank you @RaymondMwaura for the review, but as described in the PT story, by deleting a comment it should not be removed from the database, but is hidden from being shown. Thant's why its `isVisible` status has to be changed from default `true` to `false` with a `PATCH` method.